cookbook/bundles/extension.rst

```diff
@@ -119,3 +119,8 @@ Using Configuration to Change the Services
 The Extension is also the class that handles the configuration for that
 particular bundle (e.g. the configuration in ``app/config/config.yml``). To
 read more about it, see the ":doc:`/cookbook/bundles/configuration`" article.
+
+.. caution::
+
+    If you removed the default file with services definitions (i. e. ``app/config/services.yml``),
+    make sure to also remove it from the ``imports`` key in ``app/config/config.yml``.
```

Comment:
I think just before the "Using Configuration to Change the Services" section will be a much better location for this caution (as it's not related to the last section).

Please also note that we count the 72th character from the start of the line (so with indentation spaces included) and `i. e.` should be `i.e.`.

If you can do these changes, great! Otherwise, they are quite trival so we can make them during the merge process as well.